### PR TITLE
refactor(frontend): Change AxiosHooks use to don't send as a props (#64)

### PR DESCRIPTION
**Pull request by Rolly007**

### CODE REVIEW

File: frontend/src/utils/selectAxiosHook.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {
```
Comment (revodark) [inline frontend/src/utils/selectAxiosHook.js]: Maybe this file could be in the services folder, so we can have all our Axios related files in the same folder, but I'm not sure, I think that you could double check this with the team

File: frontend/src/views/MovieInfo/MovieInfo.test.js [hunk 2]
```diff
@@ -87,25 +87,27 @@ const movie = {
...
-    const useAxiosHook = () => [{ data: movie, error: false }];
+    customAxios.useAxiosOneDay = jest
+      .fn()
+      .mockReturnValue([{ error: false, data: movie, loading: false }]);
 
     await act(async () => {
       const { getByTestId, getByText } = render(
         <BrowserRouter>
-          <MovieInfo useAxiosHook={useAxiosHook} />
+          <MovieInfo id='442065' />
```
Comment (revodark) [inline frontend/src/views/MovieInfo/MovieInfo.test.js]: I think that you could double check if is necessary to pass the id as a prop

Comment (Rolly007) [reply frontend/src/views/MovieInfo/MovieInfo.test.js]: solved

File: frontend/src/utils/selectAxiosHook.js (hunk 1 above)
Comment (revodark) [reply frontend/src/utils/selectAxiosHook.js]: Since this is being modified in the PR #88 , we agreed to merge this PR with it's current state

**Pull request merged**